contracts/contracts/TACoApplication.sol

```diff
@@ -655,10 +657,18 @@ contract TACoApplication is
     }
 
     /**
-     * @notice Get all tokens delegated to the staking provider
+     * @notice Get eligible stake delegated to the staking provider
      */
```

Comment:
```suggestion
    /**  
     * @notice Returns the amount of stake that are going to be effectively  
     *         staked until the specified date. I.e: in case a deauthorization  
     *         is going to be made during this period, the returned amount will 
     *         be the staked amount minus the deauthorizing amount.  
     */  
```

Same comment that in TACoChildApplication.sol:

Reply:
✔️